### PR TITLE
CICE-14: Adding routes and directions to map

### PR DESCRIPTION
**Pull request by facuzero**
Add icons to routes but delete container indicators

### CODE REVIEW

File: pages/places/_id/place-details.vue [hunk 1]
```diff
@@ -129,6 +164,49 @@ export default {
...
+      if (box.length > 0) {
+        box[0].remove()
+      }
+      map.eachLayer(function (layer) {
+        if (layer._route) {
+          map.removeLayer(layer)
+        }
+      })
+
+      switch (param) {
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:183]: @facuzero change the case variables for something more readable and understandable, like car, foot and bicycle, anyone that read this, will not understand what this does, I'll comment where to change it.

File: pages/places/_id/place-details.vue [hunk 2]
```diff
@@ -33,10 +33,40 @@
...
+      <div class="w-11/12 md:w-4/5 my-8 overflow-hidden rounded-lg">
+        <div style="height: 30rem; display:flex; justify-content: center;">
+          <client-only>
+            <l-map id="map" ref="myMap" :zoom="12" :center="[$store.state.latitude, $store.state.longitude]" @ready="onReadyMap()">
+              <l-tile-layer url="http://{s}.tile.osm.org/{z}/{x}/{y}.png" />
+            </l-map>
+          </client-only>
+        </div>
+        <div>
+          <button class="icons" id="car" @click="onReadyMap('1')">
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:46]: @facuzero here.

File: pages/places/_id/place-details.vue [hunk 3]
```diff
@@ -33,10 +33,40 @@
...
+            <i class='fa-solid fa-car'></i>
+          </button>
+        </div>
+        <div>
+          <button class="icons" id="bike" @click="onReadyMap('2')">
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:51]: @facuzero here.

File: pages/places/_id/place-details.vue [hunk 4]
```diff
@@ -33,10 +33,40 @@
...
+            <i class="fa-solid fa-bicycle"></i>
+          </button>
+        </div>
+        <div>
+          <button class="icons" id="foot" @click="onReadyMap('3')">
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:56]: @facuzero and here.

**Review by njrzr — CHANGES_REQUESTED** [review]
@facu make these changes ASAP, to approve the PR, thanks.

File: pages/places/_id/place-details.vue (hunk 4 above)
Comment (facuzero) [reply pages/places/_id/place-details.vue:56]: Understood

**Review by njrzr — APPROVED** [review]

**Pull request merged**